tempodb/backend/s3/config.go

```diff
@@ -21,4 +21,6 @@ type Config struct {
 	SignatureV2    bool              `yaml:"signature_v2"`
 	ForcePathStyle bool              `yaml:"forcepathstyle"`
 	Tags           map[string]string `yaml:"tags"`
+	StorageClass   string            `yaml:"storage_class"`
+	UserMetadata   map[string]string `yaml:"user_metadata"`
```

Comment:
nit: I'd like to remove `user_` and make the yaml just `metadata`, to match tags and also it looks like the underlying s3 api doesn't describe it as "user"-level, maybe it's implied.   Thoughts?

Reply:
Good idea